docs/geode-native-docs/introduction/client-configurations.html.md.erb

```diff
@@ -157,7 +157,7 @@ The following libraries are external dependencies of the client library, `libgfc
 
 If you plan on using SSL in your Geode client and server deployment, you will need to download and install OpenSSL.
 
-The client requires OpenSSL 1.0.1t or later. For Windows platforms, you can use either the regular or the OpenSSL "Light" version.
+The client requires OpenSSL 1.0.2l or later. For Windows platforms, you can use either the regular or the OpenSSL "Light" version.
```

Comment:
Somehow we need to convey that it is restricted to the 1.0.2 releases of OpenSSL. It would not work for someone to install 1.1.0 despite it meeting the terms of "later".